torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2376,6 +2395,12 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                                 device_type='cuda', dtypes=[torch.cfloat, torch.cdouble],
                                 active_if=IS_WINDOWS),
                    )),
+    OpInfo('atan2',
+           dtypes=all_types_and_complex_and(torch.bool),
```

Comment:
This caused the issue. Since dtypesIfROCM is not passed it defaults to this which is incorrect. :(